src/m365/aad/commands/user/user-get.ts

```diff
@@ -50,7 +66,20 @@ class AadUserGetCommand extends GraphCommand {
 
     request
       .get(requestOptions)
-      .then((res: any): void => {
+      .then((res: any): Promise<User> => {
```

Comment:
Since we know the shape of the data that we expect from the API, let's avoid using `any` and use the expected types instead to avoid runtime errors.